sql-plugin/src/main/spark330/scala/com/nvidia/spark/rapids/shims/ParquetSchemaClipShims.scala

```diff
@@ -94,9 +94,6 @@ object ParquetSchemaClipShims {
     val typeAnnotation = parquetType.getLogicalTypeAnnotation
     val typeName = parquetType.getPrimitiveTypeName
 
-    def typeString =
-      if (typeAnnotation == null) s"$typeName" else s"$typeName ($typeAnnotation)"
-
     def typeNotImplemented() =
       TrampolineUtil.throwAnalysisException(s"Parquet type not yet supported: $parquetType")
```

Comment:
Looking at the corresponding [Spark code](https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetSchemaConverter.scala#L202-L203), I believe this is suppose to be leveraging `typeString`, e.g.:
`Parquet type not yet supported: ${typeString}`